arm-trafficmanager/2017-05-01/swagger/trafficmanager.json

```diff
@@ -0,0 +1,972 @@
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "The updated Traffic Manager profile.", 
+             "schema": { 
+               "$ref": "#/definitions/Profile" 
+             } 
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Network/trafficManagerGeographicHierarchies/default": {
+      "get": {
+        "tags": [
+          "GeographicHierarchies"
+        ],
+        "operationId": "GeographicHierarchies_GetDefault",
+        "description": "Gets the default Geographic Hierarchy used by the Geographic traffic routing method.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "The default Geographic Hierarchy.", 
+             "schema": { 
+               "$ref": "#/definitions/TrafficManagerGeographicHierarchy" 
+             } 
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DeleteOperationResult": {
+      "properties": {
+        "boolean": {
+          "type": "boolean",
+          "x-ms-client-name": "operationResult",
+          "readOnly": true,
+          "description": "The result of the operation or request."
+        }
+      },
+      "description": "The result of the request or operation."
+    },
+    "EndpointProperties": {
+      "properties": {
+        "targetResourceId": {
+          "type": "string",
+          "description": "Gets or sets the Azure Resource URI of the of the endpoint.  Not applicable to endpoints of type 'ExternalEndpoints'."
+        },
+        "target": {
+          "type": "string",
+          "description": "Gets or sets the fully-qualified DNS name of the endpoint.  Traffic Manager returns this value in DNS responses to direct traffic to this endpoint."
+        },
+        "endpointStatus": {
+          "type": "string",
+          "description": "Gets or sets the status of the endpoint..  If the endpoint is Enabled, it is probed for endpoint health and is included in the traffic routing method.  Possible values are 'Enabled' and 'Disabled'."
+        },
+        "weight": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the weight of this endpoint when using the 'Weighted' traffic routing method. Possible values are from 1 to 1000."
+        },
+        "priority": {
+          "type": "integer",
+          "format": "int64",
```

Comment:
Should be modeled with `minimum` & `maximum` like https://github.com/Azure/azure-rest-api-specs/blob/d4c1ad933ee01abf2f7c3dc0ac4034fc0e6c4317/arm-monitor/2015-04-01/examples/createOrUpdateAutoscaleSetting.json#L14 example